src/KafkaFlow.Serializer.SchemaRegistry.ConfluentJson/KafkaFlow.Serializer.SchemaRegistry.ConfluentJson.csproj

```diff
@@ -1,22 +1,20 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
-  <PropertyGroup>
-    <TargetFramework>netstandard2.0</TargetFramework>
-    <PackageId>KafkaFlow.Serializer.ConfluentJson</PackageId>
-    <Description>Json implementation for KafkaFlow serializer middleware using Confluent.SchemaRegistry.Serdes.Json package</Description>
-    <PackageIconUrl>https://raw.githubusercontent.com/Farfetch/.github/master/images/fuse-logo-128.png</PackageIconUrl>
```

Comment:
Why was PackageIconUrl removed?

Reply:
I put it in the props file. Doing this way is easier to change in the future e to not forget to add it in new projects

Reply:
nice!